src/utils/getCollectionMetadata.tsx

```diff
@@ -0,0 +1,49 @@
+import { EN_DASH } from "@/constants/chars";
+import { IMetadata, TCollectionPublicWithFamilies } from "@/types";
+import { buildConceptHierarchy } from "@/utils/buildConceptHierarchy";
+import { displayConceptHierarchy } from "@/utils/displayConceptHierarchy";
+
+export const getCollectionMetadata = (collection: TCollectionPublicWithFamilies): IMetadata[] => {
+  const collectionMetadata: IMetadata[] = [];
+
+  // Get all events from all families in the collection
+  const allEvents = collection.families.flatMap((family) => family.events);
+
+  const filingDates = allEvents
+    // TODO: confirm if events must be of type "Filing Year For Action" or "Filing Date"
```

Comment:
answer: This _might_ be the case for litigation, but could change.

Other types of collection will not have filing dates.